conjur-api.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'activesupport'
```

Comment:
Sure, but the API doesn't. It's up to activesupport's gemspec to specify which version of Ruby it requires.